tests/falco/events.json

```diff
@@ -0,0 +1,278 @@
+{
+  "events": [
```

Comment:
Out of curiosity, which Falco version did you use to generate this list?

Reply:
```
Falco version: 0.34.1
Libs version:  0.10.4
Plugin API:    2.0.0
Engine:        16
Driver:
  API version:    3.0.0
  Schema version: 2.0.0
  Default driver: 4.0.0+driver
```
Falco version: 0.34.1